src/appleseed.shaders/src/maya/as_maya_file.osl

```diff
@@ -49,26 +49,6 @@ shader as_maya_file
         string label = "Disable File Load",
         string widget = "checkBox"
     ]],
-    int in_useFrameExtension = 0
-    [[
-        string as_maya_attribute_name = "useFrameExtension",
-        string label = "Use Frame Extension",
-        string widget = "checkBox"
-    ]],
-    int in_frameExtension = 1
-    [[
-        string as_maya_attribute_name = "frameExtension",
-        string label = "Frame Extension",
-        string widget = "number",
-        int slider = 0
-    ]],
-    int in_frameOffset = 0
-    [[
-        string as_maya_attribute_name = "frameOffset",
-        string label = "Frame Offset",
-        string widget = "number",
-        int slider = 0
-    ]],
```

Comment:
This is only for the file node. Does not apply to movie or psd.
I have to check in the Maya API if there's something for the psd and movie files.